### PR TITLE
Fix log levels in GetNumericValue.

### PR DESCRIPTION
**Pull request by mfahadahmed**

### CODE REVIEW

File: OptimizelySDK/Utils/EventTagUtils.cs [hunk 1]
```diff
@@ -68,63 +68,75 @@ public static object GetRevenueValue(Dictionary<string, object> eventTags, ILogg
...
+            {
+                logMessage = "The numeric metric key is not in event tags.";
+                logLevel = LogLevel.DEBUG;
+            }
             else if (eventTags[VALUE_EVENT_METRIC_NAME] == null)
-                debugMessage = "The numeric metric key value is not defined in event tags.";
-
+            {
+                logMessage = "The numeric metric key value is not defined in event tags.";
+                logLevel = LogLevel.ERROR;
```
Comment (mikeng13) [inline OptimizelySDK/Utils/EventTagUtils.cs]: This should also be DEBUG

Comment (mfahadahmed) [reply OptimizelySDK/Utils/EventTagUtils.cs]: Done. Please let me know if the same change needs to be done for GetRevenueValue.

Comment (mikeng13) [reply OptimizelySDK/Utils/EventTagUtils.cs]: Yes, please also make the change for `GetRevenueValue`

**Comment by mikeng13** [comment]
Fixes #85

**Pull request merged**